modules/org.kie.kogito.app.builder/install.sh

```diff
@@ -2,4 +2,4 @@
 set -e
 cd /workspace
 
-CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
\ No newline at end of file
+CGO_ENABLED=0 GO111MODULE=on go build -a -o manager main.go
```

Comment:
probably it needs to be adjusted on the other places as well:

- the kogito-cli will not be part of it, but might be good to remove it from there as well (go-build-cli and go-install-cli scripts)
- modules/org.kie.kogito.app.buildProfilingOperator/install.sh
- modules/org.kie.kogito.rhpam.builder/install.sh
-